koku/masu/database/azure_report_db_accessor.py

```diff
@@ -442,6 +456,34 @@ def check_for_matching_enabled_keys(self):
                 return False
         return True
 
+    def _get_matched_tags_strings(self, bill_id, azure_provider_uuid, ocp_provider_uuid, start_date, end_date):
```

Comment:
This functionality seems really similar across the hyperscalers and may be better suited in the base class.

Reply:
It does but it means refactoring a lot of other stuff to which I'm not sure I want to tackle right now. Maybe I will look at it again while tests are running.

Reply:
If you feel like it would be a big lift to move it to the base class, lets just move forward with this